doc/BaseRegTest.rst

```diff
@@ -1,12 +1,8 @@
-.. _BaseRegTest:
```

Comment:
Why remove the targets?

Reply:
I figured that people can just reference the class directly via `:class:BaseRegTest` or reference the page via the `:doc:` directive, so these targets are unnecessary.